### ISSUE TITLE
Missing role="tab" attribute in designer tabs markup

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-creator#963]
See the https://surveyjs.answerdesk.io/ticket/details/t5238/accessibility-ensure-elements-with-aria-role-that-require-child-role-contain-them thread for more details
